proto/o5/dempe/v1/service.proto

```diff
@@ -44,37 +53,41 @@ message GetMessageRequest {
 
 message CapturedMessage {
   DeadMessage cause = 1;
-
 }
 
 message GetMessageResponse {
   CapturedMessage message = 1;
-  repeated MessageAction actions = 2;
+
+  message GetMessageAction {
+    MessageAction action = 1;
+    google.protobuf.Timestamp timestamp = 2;
+    o5.auth.v1.Actor actor = 3;
+  }
+  repeated GetMessageAction actions = 2;
 }
 
 message MessageAction {
+  string id = 1 [
+    (buf.validate.field).string.uuid = true
+  ];
```

Comment:
I realized, this would actually be the action's ID, not the message ID. So I'm not sure how necessary this one is. Possibly should be removed.

Reply:
I was thinking it would make sense for the action to have an id, because it could appear in an "audit log." I assume there's a use case for showing actions on a message, because we're storing actor data for the actions. To that note, I'd imagine we should add another endpoint for a list of actions for a message.

Reply:
I haven't even considered storing an audit log. Seems like a good idea!